### ISSUE TITLE
jitlog parsing is terrible, undocumented and broken

### ISSUE THREAD

**Issue by fijal** [vmprof/vmprof-python#96]
There was a requirement to have a tool that would turn jitlog into the old text format in case things go wrong. It plain does not work for several reasons:

* the docs for jit logs does not mention -m jitlog
* if you actually run it, it complains that None has no attribute name
* even if it did, it comes with only -i <x> -q foo (undocumented) program-name (why?)
* even if it worked, it does not because it fails to parse some stuff. The point of as-good-as-old-tool is to not parse inconsistencies but print it
* even if it did, it explodes with NotImplementedError("address conflicting"). All fine, but I want to see my traces
* then it explodes with NameError on something https://paste.pound-python.org/show/dtg9qEltfQgUXoj4FEtn/

Overall it really does not work and is untested up to the point where there are NameErrors

**Comment by planrich** [comment]
Yep, we wrote a very minimal pretty printer in Bucharest. But, it did not make it through all the changes that came after. I'm sorry that it does not work currently (at all). The reason why it is not documented is that it is not finished.

I also removed 'terrible' from the title.
